mindsdb/integrations/handlers/fireworks_handler/config.py

```diff
@@ -0,0 +1,8 @@
+supported_task = {
```

Comment:
Maybe change this to something more meaningfull too as: conversational or image mode?